coreneuron/CMakeLists.txt

```diff
@@ -170,7 +179,7 @@ target_compile_options(coreneuron
                        PRIVATE $<$<COMPILE_LANGUAGE:CXX>:${CORENEURON_CXX_WARNING_SUPPRESSIONS}>)
 
 # need to have _kinderiv.h for mod2c generated files and nrnivmodl-core and nmodl building
-add_dependencies(coreneuron kin_deriv_header nrnivmodl-core)
+add_dependencies(coreneuron kin_deriv_header corenrn_mpi nrnivmodl-core)
```

Comment:
Is this needed? Looks redundant with `$<TARGET_OBJECTS:corenrn_mpi>` above.

Reply:
yup, your are right!